doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,895 @@
+    This configuration is specified in the :file:`ncs/nrf/subsys/bluetooth/mesh/Kconfig` file.
+
+See `Bluetooth mesh samples`_ for the list of changes in the Bluetooth mesh samples.
+
+Matter
+------
+
+* Added:
+
+  * Support for the :ref:`ug_matter_configuring_optional_persistent_subscriptions` feature.
+  * The Matter Nordic UART Service (NUS) feature to the :ref:`matter_lock_sample` sample.
+    This feature allows using Nordic UART Service to control the device remotely through Bluetooth LE and adding custom text commands to a Matter sample.
+    The Matter NUS implementation allows controlling the device regardless of whether the device is connected to a Matter network or not.
+    The feature is dedicated for the Matter over Thread solution.
+  * Documentation page about :ref:`ug_matter_device_configuring_cd`.
+  * Matter SDK fork :ref:`documentation pages <matter_index>` with the page about CHIP Certificate Tool.
+  * Documentation page about :ref:`ug_matter_device_adding_bt_services`.
+
+* Updated:
+
+  * The :ref:`ug_matter` protocol page with a table that lists compatibility versions for the |NCS|, the Matter SDK, and the Matter specification.
+  * The :ref:`ug_matter_tools` page with installation instructions for the ZAP tool, moved from the :ref:`ug_matter_creating_accessory` page.
+  * The :ref:`ug_matter_tools` page with information about CHIP Tool, CHIP Certificate Tool, and the Spake2+ Python tool.
+  * The :ref:`ug_matter_device_low_power_configuration` page with information about sleepy active threshold parameter configuration.
+  * The default number of user RTC channels on the nRF5340 SoC's network core from 2 to 3 (the platform default) to fix the CSL transmitter feature on Matter over Thread devices acting as Thread routers.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.1.0.1`` tag.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added:
+
+ * ``SLEEPY_ACTIVE_THRESHOLD`` parameter that makes the Matter sleepy device stay awake for a specified amount of time after network activity.
+
+* Updated:
+
+  * The factory data generation script with the feature for generating the onboarding code.
+    You can now use the factory data script to generate a manual pairing code and a QR Code that are required to commission a Matter-enabled device over Bluetooth LE.
+    Generated onboarding codes should be put on the device's package or on the device itself.
+    For details, see the Generating onboarding codes section on the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
+  * The Basic Information cluster with device finish and device color attributes and added the related entries in factory data set.
+
+Thread
+------
+
+|no_changes_yet_note|
```

Comment:
@peknis 

can we add here a following sentence:

```
Threat in impacted by CVE-2023-2626. Due to this issue, |NCS| v2.4.0 will not undergo the certification process, and is not intended to be used in final Thread products.
```

Reply:
@bpienk fyi

Reply:
Sure. @umapraseeda is in charge here. I don't think we need to link to that known issue (that we added to my PR) as the contents is pretty much the same.

Reply:
Added this as agreed with @bpienk offline.